README.rst

```diff
@@ -105,3 +105,13 @@ with the same interface as the popular `HappyBase
 <https://happybase.readthedocs.io/en/latest/>`__ library. Unlike HappyBase,
 ``google-cloud-happybase`` uses ``google-cloud-bigtable`` under the covers,
 rather than Apache HBase.
+
+``experimental async data client``
+----------------------------------
+
+`v2.23.0` includes a preview release of a new data client, accessible at `google.cloud.bigtable.data.BigtableDataClientAsync`.
+
```

Comment:
I think we can point this README to a new README file under _async, and provide some examples there.

Reply:
Good point about the secondary README, I added another one in the `/data` folder

But we discussed offline adding the usage examples later, so for now it just includes some of the same content as this one. Let me know what you think

Reply:
Yup let's update examples later!

Reply:
Can we move this section to the top of the document? (or at least above HappyBase) I don't think we want to advertise happybase anymore

Reply:
Sure, moved to the top and removed the happybase section

I also added feedback links